server/src/main/java/umm3601/Server.java

```diff
@@ -10,6 +10,12 @@
 public class Server {
     public static void main(String[] args) throws IOException {
 
+        // This users looks in the folder `public` for the static web artifacts,
+        // which includes all the HTML, CSS, and JS files generated by the Angular
+        // build. This `public` directory _must_ be somwhere in the classpath;
+        // a problem which is resolved in `server/build.gradle`.
+        staticFiles.location("/public");
```

Comment:
Hmm... Have you tried increasing the logging level of the spark server? I'm not sure exactly how to do that, but it's certainly possible. First, maybe try using ``gradlew run --info`` ?